.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Is this vscode specific metadata file which has environment specific port number info supposed to be part of the project?

Reply:
We had this file in the Appsody `nodejs` templates, we thought it might be easier for the user as they wouldn't have to create it themselves. But i do see where you are coming from and we should keep it uniform across all stack start project so we've removed it

Reply:
This is interesting one..  User will need to know `reoteRoot` path to eb able to setup this themselves. 
@elsony do we document this somewhere today? I remember there was a field added to 2.0 spec for pointing to stack documentation. Should we look to use that for capturing such info?

Reply:
The attach config doesn't actually require the `remoteRoot` path, it is enough to give it the correct port, I was just trying to be "helpful". For example, the following debugging config also works for me: 
```
    "version": "0.2.0",
    "configurations": [
        {
            "type": "node",
            "request": "attach",
            "name": "Attach Program",
            "skipFiles": [
                "<node_internals>/**"
            ],
            "port": 5858,
        }
    ]
}
```

Reply:
That config is largely generated by vscode, I just changed the request from "launch"  to "attach" and added the port

Reply:
@neeraj-laad  For your question on doc, any stack/devfile specific doc should go to the website under the metadata on a devfile.  For info specific to the starter project, it should go to the README file in the project.